### PR TITLE
Bugfix104/vcf reverse strand

### PR DESCRIPTION
**Pull request by at7**
Fix for [ENSVAR-4003]( https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-4003). All the information is in the ticket

### CODE REVIEW

**Comment by dglemos** [comment]
The consequence type is fixed. The bug was in this page: http://ves-hx2-76.ebi.ac.uk:7040/Oreochromis_niloticus/Gene/Variation_Gene/Table?db=core;g=ENSONIG00000000975;r=LG19:18874597-18906373;source=PRJEB38548;v=LG19:18886255:C_T:PRJEB38548;vdb=variation;vf=LG19:18886255:C_T:PRJEB38548

**Review by dglemos — APPROVED** [review]

**Review by helensch — COMMENTED** [review]
Looks good. Only minor documentation comments.
I have checked using mink variants see ENSVAR-3909
Are there any other methods in VCFCollection.pm that need to have the slice changed when the slice is on the reverse strand e.g get_all_SampleGenotypeFeatures_by_Slice?

**Comment by at7** [comment]
Thank you both for checking and your feedback. I addressed everything and created a ticket for checking if reverse strands cause problems for other methods in the VCFCollection module. ENSVAR-4049

File: modules/Bio/EnsEMBL/Variation/VCFCollection.pm [hunk 1]
```diff
@@ -433,11 +433,19 @@ sub get_all_VariationFeatures_by_Slice {
   my $self = shift;
   my $slice = shift;
   my $dont_fetch_vf_overlaps = shift;
-  
+  # Variants from a VCF file are always on the forward strand. If a slice on the reverse strand is passed to the sub we
+  # create the same slice but on the forward strand using fetch_by_region and attach it to the VCFVariationFeature object.
+  # Attaching the slice on the reverse strand to the VCFVariationFeature object would cause problems when
+  # calculating the consequence for an overlapping transcript which is also on the reverse strand. The variant alleles would not be reverse completed
```
Comment (helensch) [inline modules/Bio/EnsEMBL/Variation/VCFCollection.pm:439]: Small typo - completed should be complemented

**Review by helensch — APPROVED** [review]

**Comment by at7** [comment]
@dglemos could you please merge the PR? Thank you

**Comment by dglemos** [comment]
Merged into release/104 and master

**Pull request merged**